locales/en-US/strings.properties

```diff
@@ -205,3 +205,7 @@ firstrun_privacy_notice=Privacy Notice
 
 firstrun_continue_to_login=Continue
 firstrun_skip_login=Skip this step
+
+# LOCALIZATION NOTE (context_menu_title):
+# context_menu_title is shown as tooltip
```

Comment:
Repeating the string id is redundant as it's already in the localization note, and the comment can start on the previous line. The comment should also mention that this is an action (as "open" could be confused as an adjective describing that the menu is open). So maybe just:

`# LOCALIZATION NOTE (context_menu_title): Action tooltip to open a context menu`

Reply:
Thanks @Mardak for this feedback. Will make the change right away 👍